libmscore/element.h

```diff
@@ -20,6 +20,8 @@
 
 namespace Ms {
 
+// CONTROL_MODIFIER is to be used when handling text editing.
+// Qt::ControlModifier (or EditData::control()) is to be used in all other cases.
```

Comment:
Maybe creating one more method like `controlTextEdit()` will do its job to avoid direct comparisons in the code? Also, the method name will be self explanative, so no need for the comments.